dockstore-webservice/src/main/java/io/dockstore/webservice/core/ParsedInformation.java

```diff
@@ -0,0 +1,51 @@
+package io.dockstore.webservice.core;
+
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+
+import io.dockstore.common.DescriptorLanguage;
+
+/**
+ * This is for information gained after parsing the workflow with a language parser (WDLHandler, CWLHandler, etc)
+ *
+ * Putting DescriptorLanguage here instead of using a map with DescriptorLanguage because there may be cases where the language is
+ * irrelevant. For example, workflows only have one language.
+ */
+@Entity
+public class ParsedInformation {
+    @Id
+    @GeneratedValue
+    private Long id;
```

Comment:
Might become more clear further down, but since this only is two booleans, it might be worth looking into making this embedded https://www.baeldung.com/jpa-embedded-embeddable so you can still interact with this as an object but it has no independent id and/or table from version metadata

Reply:
First thing I tried, couldn't do it because it's OneToMany instead of OneToOne.  See this commit on what it originally was https://github.com/dockstore/dockstore/pull/3692/commits/79e0078318aa359437183ce04f25e99620f6989a